scenarios/15a_The_Preparations.cfg

```diff
@@ -140,15 +150,46 @@
 
     [event]
         name=start
-        {MSG_Gwidle _"...so, you will take twelve of your best troops and search for this great mage, to try to convince him to join our cause. "}
-        {MSGm_Gawen _"I understand. My twelve best soldiers. When should I go?"}
+#ifdef EASY
+        #po: EASY difficulty, use most explicit/obvious version of wording for players like me who don't pay attention:
+        {MSG_Gwidle _"...so, you will take eighteen of your very best troops and search for this great mage, Deorien, to try to convince him to join our cause."}
+        {MSGm_Gawen _"I understand. My eighteen very best, most experienced soldiers. When should I go?"}
+#endif
+#ifdef NORMAL
+        #po: NORMAL difficulty, keep this wording closest to what it was before the difficulty split:
+        {MSG_Gwidle _"...so, you will take fifteen of your best troops and search for this great mage, to try to convince him to join our cause."}
+        {MSGm_Gawen _"I understand. My fifteen best soldiers. When should I go?"}
+#endif
+#ifdef HARD
+        #po: HARD difficulty, somewhat vaguer than the other two:
+        {MSG_Gwidle _"...so, you'll take twelve of your troops and search for that mage, to try to convince him to join our cause."}
+        {MSGm_Gawen _"I understand. Twelve soldiers. When should I go?"}
+#endif
         {MSG_Gwidle _"Does that mean that you agree?"}
         {MSGm_Gawen _"Of course."}
+#ifdef HARD
+        #po: HARD difficulty:
+        {MSG_Gwidle _"I think you should start your quest after the thaw. After all, you'll need some time to pick soldiers and train them."}
```

Comment:
I think it's not really worth it for translators to split the normal and hard difficulty ones for wording.

Reply:
ok I'll change this part later; I should be going to bed for tonight now, though...

Reply:
ok reverted in [6ee22b6](https://github.com/nemaara/A_New_Order/pull/43/commits/6ee22b68d86284ca003102fedd62de0c9e22ce57)